pkg/agent/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -81,11 +84,13 @@ func NewNetworkPolicyController(antreaClientGetter agent.AntreaClientProvider,
 	ofClient openflow.Client,
 	ifaceStore interfacestore.InterfaceStore,
 	nodeName string,
-	podUpdates <-chan v1beta1.PodReference) *Controller {
+	podUpdates <-chan v1beta1.PodReference,
+	antreaPolicyEnabled bool) *Controller {
 	c := &Controller{
 		antreaClientProvider: antreaClientGetter,
 		queue:                workqueue.NewNamedRateLimitingQueue(workqueue.NewItemExponentialFailureRateLimiter(minRetryDelay, maxRetryDelay), "networkpolicyrule"),
 		reconciler:           newReconciler(ofClient, ifaceStore),
+		antreaPolicyEnabled:  antreaPolicyEnabled,
```

Comment:
missing change in cmd/antrea-agent/agent.go to populate this field?

Reply:
Oh, I forgot to add agent.go. Updated.